src/components/common/Filter.tsx

```diff
@@ -144,11 +144,11 @@ export default function Filter({
           <span className="flex-1 pb-16 text-body1/26">이상부터</span>
         </div>
 
-        <div className="flex gap-4">
-          <Button solid={false} className="w-80" onClick={handleReset}>
+        <div className="fixed right-0 bottom-0 left-0 flex w-full gap-8 border-t border-gray-20 bg-white px-12 py-16 md:static md:border-none md:p-0">
+          <Button solid={false} className="w-82" onClick={handleReset}>
```

Comment:
```suggestion
          <Button solid={false} className="w-82 flex-1" onClick={handleReset}>
```
💬 이런 식으로 추가해야 아래 버튼의 여백이 없을 것 같습니다~